tasks/libs/common/utils.py

```diff
@@ -72,6 +73,22 @@ def bin_name(name):
     return name
 
 
+def get_distro():
+    """
+    Get the distro name. Windows and Darwin stays the same.
+    Linux is the only one that needs to be determined using the /etc/os-release file.
+    """
+    system = platform.system()
+    arch = platform.machine()
+    if system == 'Linux':
+        if os.path.isfile('/etc/os-release'):
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** Can you rewrite the condition with an `and` ?‏

Reply:
Nice catch, fixed in [278b761](https://github.com/DataDog/datadog-agent/pull/25652/commits/278b761e001f22ad3bd55f0aaf26eac07a034925), thanks !